src/logger.py

```diff
@@ -26,18 +30,26 @@ class Logger:
         self.logger = logger if logger else Logger(show_message=False).logger
 
     """
-    def __init__(self, log_level=logging.INFO, 
-                 logfile="logs/logs.txt",
+    def __init__(self,
+                 logger_name="default",
+                 log_level=logging.INFO, 
+                 logfile="logs/ols.logs",
                  show_message=False):
         msg=f"""
         ############################################################################
-        Set LOG_LEVEL environment variable (e.g., INFO, DEBUG) to cocntrol verbosity
+        Set LOG_LEVEL or LOG_LEVEL_CONSOLE environment variable (e.g., INFO, DEBUG) 
+        to control general logging verbosity or console specific logging level
         ############################################################################
         """
         if show_message: print(msg)
 
+        self.logger_name=logger_name
         self.log_level=os.getenv("LOG_LEVEL", log_level)
+        self.log_level_console=os.getenv("LOG_LEVEL_CONSOLE", self.log_level)
         self.logfile=logfile
+        self.logfile_maxSize=1048576*100
+        self.logfile_backupCount=3
```

Comment:
why don't we want to expose these as parameters?

Reply:
We should